test/IECoreScene/MeshAlgoSegmentTest.py

```diff
@@ -184,17 +184,21 @@ def testSegmentSubset( self ) :
 		self.assertEqual( segments[0]["s"].data, IECore.StringVectorData( ["b"] ) )
 
 
-	def testSegmentUsingIndexedPrimitiveVariable( self ) :
+	# def testSegmentUsingIndexedPrimitiveVariable( self ) :
```

Comment:
This test is reinstated in a later commit. 

Perhaps  I need to add more segment tests for indexed primvars for Points & Curves? There is already some test coverage in the DeleteFaces / Curves / Points already though.